packages/@dcl/ecs/src/engine/input.ts

```diff
@@ -99,15 +99,14 @@ const InternalInputStateSchema = {
   )
 }
 
-const InternalInputStateComponentId = 1500
 const TimestampUpdateSystemPriority = 1 << 20
 const ButtonStateUpdateSystemPriority = 0
 
 export function createInputSystem(engine: IEngine): IInputSystem {
   const PointerEventsResult = components.PointerEventsResult(engine)
   const InternalInputStateComponent = engine.defineComponent(
-    InternalInputStateSchema,
-    InternalInputStateComponentId
+    '@dcl/sdk/InternalInputStateSchema',
```

Comment:
should we use a unique convention like `dcl::sdk::internalInputState` like `core::Transform` ?

Reply:
I think it makes sense to get a convention here, nevertheless, this one lives in the same realm as the ones created and used from the scenes and libraries. It is to all extent an opaque identifier. Is it OK if we change it once we get agreement on the convention and merge it as it is?